src/store/getters.ts

```diff
@@ -505,9 +505,7 @@ export const getEntityTypesXPRO = (state: StateIF): EntityI[] => {
 }
 
 export const getShowXproJurisdiction = (state: StateIF): boolean => {
-  if (getLocation(state) !== Location.BC) return true
-  if (getRequestActionCd(state) === RequestCode.MVE) return true
```

Comment:
Why is it OK to remove this MVE check? Should this be commented in the ticket as something for QA to verify?

Reply:
Never mind - I read your description of changes again. Looks fine.

Reply:
I confirmed with Linda this should never have been required in the EDIT flow for a MVE request. 

I believe, the initial implementation of MVE requests were thought of as XPRO but the flow changed to a BC request when creating a new request. I believe this was left over from that change.

Reply:
We also verified the same behaviour in Name Examination and NRO.

Reply:
Will add a comment to the ticket for QA, as to what to look for.